staging/src/k8s.io/code-generator/cmd/validation-gen/validation.go

```diff
@@ -105,7 +108,7 @@ func (g *genValidations) isOtherPackage(pkg string) bool {
 func (g *genValidations) Init(c *generator.Context, w io.Writer) error {
 	sw := generator.NewSnippetWriter(w, c, "$", "$")
```

Comment:
BTW, @thockin do you have a preference between `{...}` and `$..$` for template variable substitution?  I'd be happy to make switch if you like `{..}` better.

Reply:
I don't have a strong pref, but since `{...}` is valid syntax in Go, maybe not - do we do that anywehre else?  I always assumed the `$...$` was not valid Go syntax, it makes the `sw.Do` string literals more obvious.  I don't LOVE it, but it's not ambiguous.  And it's used everywhere else.

Convince me different is better?

Reply:
I don't feel strongly. I noticed a mix of use in the generators the other day so figured I'd ask.  The observation that `$` is not valid go seems compelling. Let's stick with it.